.github/workflows/update-nextcloud-ocp-matrix.yml

```
@@ -0,0 +1,78 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+name: Update nextcloud/ocp
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '5 2 * * 0'
+
+jobs:
+  update-nextcloud-ocp:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        branches: ['master']
+        target: ['stable22']
```

I think that we don't do support to 22 version.

```suggestion
        target: ['stable24', 'stable25', 'stable26', 'master']
```

---

Yes, we do. For now the app still supports Nextcloud 22: https://github.com/nextcloud/monthly_status_email/blob/47fb5193fe5d2a0df93bed54459cea9a7518ac54/appinfo/info.xml#L17

If we want to remove support for ancient Nextcloud versions, let's discuss this in a separate PR/issue, no?

---

Also, target in this case should only contain only one entry to my understanding, the `dev-<target>` version that we aim to keep `nextcloud/ocp` updated to, no? See https://github.com/nextcloud/.github/blob/master/workflow-templates/update-nextcloud-ocp-matrix.yml#L44

---

Wow! Will be good to split the branches in all stable branches and use the backport bot to send the changes to all stables branches. We can do this in a follow up and separated PR too.